modules/extensions-xml-support/src/main/java/org/mule/runtime/extension/internal/loader/XmlExtensionLoaderDelegate.java

```diff
@@ -249,6 +243,12 @@ private void loadModuleExtension(ExtensionDeclarer declarer, ComponentModel modu
     }
   }
 
+  private XmlDslModel getXmlDslModel(ComponentModel moduleModel, String name, String version) {
```

Comment:
just read identical code in JavaXmlDeclarationEnricher.java. Doesn't look like doing the same here is necessary at all